src/RadForce/test_radiation_force.cpp

```diff
@@ -132,6 +153,13 @@ template <> void RadhydroSimulation<TubeProblem>::setInitialConditionsOnGrid(quo
 	auto const &Mach_ptr = Mach_arr_g.dataPtr();
 	int x_size = static_cast<int>(x_arr_g.size());
 
+	// CCH: calculate radEnergyFractions
```

Comment:
As a matter of style, we haven't added initials indicating the developer who has written the code, since it's always available with `git blame`.

Reply:
All resolved.